CHANGELOG.md

```diff
@@ -35,6 +35,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 * #1108 OIDC: Fix `validate_bearer_token()` to properly set `request.scopes` to the list of granted scopes.
+* Miscellaneous packaging issues.
```

Comment:
```suggestion
```
Probably not necessary to explicitly say this as it doesn't change the user-visible functionality.

Reply:
@n2ygk Fixed.